app/styles/app.css

```diff
@@ -6,6 +6,7 @@
 @import 'local-navbar.css';
 @import 'profile.css';
 @import 'get-started.css';
+@import "user-status.css";
```

Comment:
Why are the quotes different from above?

Reply:
Fixing it